app/build.gradle

```diff
@@ -16,7 +16,7 @@ buildscript {
   }
 
   dependencies {
-    classpath "com.android.tools.build:gradle:$androidGradlePluginVersion"
+    classpath "com.android.tools.build:gradle:3.4.0"
```

Comment:
Is there a reason this has changed?

Reply:
I think this change happened because I'd updated Gradle while working on the feature. I'll update the branch :+1: